cmd/server/server.go

```diff
@@ -292,12 +291,6 @@ func Initialize(l log.Logger, conf config.Optimus) error {
 	jobCompiler := compiler.NewCompiler(conf.Server.IngressHost)
 	// init default scheduler
 	switch conf.Scheduler.Name {
-	case "airflow":
```

Comment:
Should airflow also be mapped to airflow2, as it is the current version of airflow. ?

Reply:
agree on this, making the change